### ISSUE TITLE
faster?

### ISSUE THREAD

**Issue by jbenet** [ipfs/fs-stress-test#1]
right now the second test hangs for a long time in `ipfs add`. Looks to be hashing (pegs 1 cpu). Maybe
- concurrent add wil help https://github.com/ipfs/go-ipfs/pull/1324 (cc @barnacs)
- should use more than 1cpu (cpu seems always to be below 100%)

**Comment by jbenet** [comment]
cc @whyrusleeping

**Comment by whyrusleeping** [comment]
@jbenet we do use more than one CPU: https://github.com/ipfs/go-ipfs/blob/master/cmd/ipfs/main.go#L60

**Comment by jbenet** [comment]
@whyrusleeping maybe not being used in the add we have now?

**Comment by whyrusleeping** [comment]
Yeah, our current add call is very serial. If the current concurrent PR doesnt land soon i'll go ahead and implement that.
